reference/checkouts.v3.yml

```
@@ -1747,33 +1810,10 @@ parameters:
     enum:
       - consignments.available_shipping_options
 securityDefinitions:
-  X-Auth-Token:
+  Checkouts:
+    name: API Key
```

based on the endpoints, this looks like a store-management API. if that's true, the OAuth stuff needs to go back in, and the access_token gets passed as the X-Auth-Token.

```suggestion
    name: X-Auth-Token
```